server/db/models.py

```diff
@@ -113,6 +115,14 @@
     Column("allow_override", Boolean)
 )
 
+game_join_log = Table(
+    "game_join_log", metadata,
+    Column("id",         BigInteger, primary_key=True),
+    Column("player_id",  Integer,    ForeignKey("login.id"),      nullable=False),
+    Column("game_id",    Integer,    ForeignKey("game_stats.id"), nullable=False),
+    Column("created_at", TIMESTAMP,  nullable=False, server_default=func.now()),
```

Comment:
I think the `server_default` is actually irrelevant since we're never using sqlalchemy to actually create the database tables. Looks like in our other models we've been just doing this:

```
    Column("create_time",   TIMESTAMP,  nullable=False),
    Column("update_time", TIMESTAMP, nullable=False),
```

You can also always just leave the field off completely if we don't need to ever read from it or include it in an insert statement.

Do we really want to use `created_at` rather than `create_time` like the rest of our tables?

Reply:
I don't know how SQLalchemy handles it. In Hibernate I have to declare it as non-insertable so it doesn't try to fill null on insert.

Will re-check on the column name.